src/lib/keyboard-shortcut/keyboard-shortcut-foundation.ts

```diff
@@ -82,6 +84,7 @@ export class KeyboardShortcutFoundation implements IKeyboardShortcutFoundation {
         evt.preventDefault();
       }
       this._adapter.emitHostEvent(KEYBOARD_SHORTCUT_CONSTANTS.events.ACTIVATE, evt);
+      this._activateCallback?.(evt);
```

Comment:
It may be worth invoking the callback with `.call(null, evt)` or `.call({}, evt)` to ensure the internal context isn't provided to the consumer.